KthSmallestElementInASortedMatrix.py

```diff
@@ -0,0 +1,58 @@
+class Solution(object):
+    def kthSmallest(self, matrix, k):
+        """
+        :type matrix: List[List[int]]
+        :type k: int
+        :rtype: int
+        """
+        if not matrix:
+            return 0
+        m = len(matrix)
+        n = len(matrix[0])
+        heap = []
+        for i in range(m):
+            self.heapAdd(heap, (matrix[i][0], i, 0))
+        while k > 1:
+            minItem = self.heapPop(heap)
```

Comment:
You can do
`_, x, y = self.heapPop(heap)`

Reply:
sure, will do